sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents_and_read_table_cells.py

```diff
@@ -0,0 +1,87 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_custom_documents_and_read_table_cells.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze a document with a custom extraction
+    built model and read data from table cells. The document must be of the same
+    type as the documents the custom model was built on. To learn how to build
+    your own models, look at sample_build_model.py.
+
+USAGE:
+    sample_analyze_custom_documents_and_read_table_cells.py
+
+    Set the environment variables with your own values before running the sample:
+    1) DOCUMENTINTELLIGENCE_ENDPOINT - the endpoint to your Document Intelligence resource.
+    2) DOCUMENTINTELLIGENCE_API_KEY - your Document Intelligence API key.
+    3) CUSTOM_BUILT_MODEL_ID - the ID of your custom built model.
+        -OR-
+       DOCUMENTINTELLIGENCE_STORAGE_CONTAINER_SAS_URL - The shared access signature (SAS) Url of your Azure Blob Storage container with your training files.
+       A model will be built and used to run the sample.
+"""
+
+
+def analyze_custom_documents_and_read_table_cells():
+    import os
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.formrecognizer import DocumentAnalysisClient
+
+    endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+    key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+    model_id = os.environ["CUSTOM_BUILT_MODEL_ID"]
+    docStorageUrl = os.environ["DOCUMENTINTELLIGENCE_STORAGE_CONTAINER_SAS_URL"]
+
+    document_analysis_client = DocumentAnalysisClient(
+        endpoint=endpoint, credential=AzureKeyCredential(key)
+    )
+
+    # Make sure your document's type is included in the list of document types the custom model can analyze
+    poller = document_analysis_client.begin_analyze_document_from_url(
+        model_id, docStorageUrl
+    )
+    result = poller.result()
+
+    for i, table in enumerate(result.tables):
+        print("\nTable {} can be found on page:".format(i + 1))
+        for region in table.bounding_regions:
+            print("...{}".format(i + 1, region.page_number))
+        for cell in table.cells:
+            print(
+                "...Cell[{}][{}] has content '{}'".format(
+                    cell.row_index, cell.column_index, cell.content.encode("utf-8")
+                )
+            )
+
+    print("----------------------------------------")
```

Comment:
Same, should we directly update in `sample_analyze_custom_documents`?

Reply:
optimized, thx